pyproject.toml

```diff
@@ -38,4 +44,4 @@ docstring-convention = "google"
 # Check for properly formatted copyright header in each file
 copyright-check = "True"
 copyright-author = "Canonical Ltd."
-copyright-regexp = "Copyright\\s\\d{4}([-,]\\d{4})*\\s+%(author)s"
\ No newline at end of file
+copyright-regexp = "Copyright\\s\\d{4}([-,]\\d{4})*\\s+%(author)s"
```

Comment:
Do you think there is a way to move this regex to the operator workflows? Maybe something like we did for the woke tool?

Reply:
Yes, good point. Maybe we can file a ticket for this, what do you think?

Reply:
Created a ticket